src/com/content-notifications/notification.js

```diff
@@ -1,9 +1,42 @@
 import {h, Component}	 				from 'preact/preact';
-
 import NavLink 							from 'com/nav-link/link';
-
 import SVGIcon 							from 'com/svg-icon/icon';
 
+
+export const isNotifactionType = (notification, notificationType) => {
```

Comment:
All of these - `Notification` rather than `Notifaction`

Reply:
![notifaction-notifactions-everywhere](https://user-images.githubusercontent.com/1570313/33732377-536502a4-db54-11e7-9c70-7d34d8b231dd.jpg)

Sorry, I couldn't resist. I would recommend checking the entire changeset for `notifaction`. They're everwhere, not just here.

Reply:
renamed all